### PR TITLE
Feature/v1 landing page

### PR DESCRIPTION
**Pull request by ramenhog**
### Purpose
Create landing page for refine.bio

Closes https://github.com/AlexsLemonade/refinebio/issues/119

### Screenshots
refine bio-desktop
refine bio-mobile

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+*.DS_Store
```
Comment (dongbohu) [inline .gitignore]: Need a new line char.

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+				<a href="https://twitter.com/CancerDataLab" target="_blank">
+					<img class="social-links__icon" src="images/twitter.svg" alt="Twitter" />
+				</a>
+				<a href="https://github.com/AlexsLemonade/refinebio" target="_blank">
+					<img class="social-links__icon" src="images/github.svg" alt="Github" />
+				</a>
+				<a href="https://gitter.im/refinebio/" target="_blank">
+					<img class="social-links__icon" src="images/gitter.svg" alt="Gitter" />
+				</a>
+				<p class="social-links__credit">A project of the Childhood Cancer Data Lab, an Alex's Lemonade Stand Foundation initiative.</p>
```
Comment (dongbohu) [inline public/index.html]: Is it better to change the line into:
```... A project of the Childhood Cancer Data Lab (CCDL), an Alex ...```
I am afraid that the term `CCDL` may be used everywhere later.

**Review by dongbohu — COMMENTED** [review]
Have some minor comments. (I skipped all svg files by the way).

**Comment by ramenhog** [comment]
Addressed those fixes. Back to you @dongbohu thanks!

**Review by dongbohu — APPROVED** [review]
Looks good!

**Pull request merged**